fe/fe-core/src/main/java/com/starrocks/catalog/StructType.java

```diff
@@ -48,9 +48,9 @@ public class StructType extends Type {
 
     private static final Logger LOG = LogManager.getLogger(StructType.class);
 
-    private final HashMap<String, StructField> fieldMap = Maps.newHashMap();
+    private HashMap<String, StructField> fieldMap = Maps.newHashMap();
```

Comment:
why remove `final`?

Reply:
> why remove `final`?

Because we will update `fieldMap` during add/drop field.

Reply:
`map add or remove` can use `final`

Reply:
> `map add or remove` can use `final`

done.